app/controllers/users_controller.rb

```diff
@@ -18,20 +18,19 @@ class UsersController < ApplicationController
   before_filter :return_here, :only => [:index, :show, :relationships, :dashboard, :curation]
   before_filter :before_edit, only: [:edit, :edit_after_auth]
   
-  MOBILIZED = [:show, :dashboard, :new, :create]
+  MOBILIZED = [:show, :new, :create]
   before_filter :unmobilized, :except => MOBILIZED
   before_filter :mobilized, :only => MOBILIZED
 
   protect_from_forgery unless: -> {
     request.parameters[:action] == "search" && request.format.json? }
 
-  caches_action :dashboard,
```

Comment:
yes - just so that the onboarding material and the sidebar stuff (e.g. announcements) aren't cached